client/spec/AnalystRatings.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import AnalystRatings from '../Components/AnalystRatings.jsx';
+
+Enzyme.configure({adapter: new Adapter()});
```

Comment:
If you make a jest.setup.js file and point to it from your jest.config.js file, you don't need to call the Adaptor every time!